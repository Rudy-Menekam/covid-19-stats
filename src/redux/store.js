import { configureStore } from '@reduxjs/toolkit';
import categoriesReducer from './categoriesSlice';

const store = configureStore({
  reducer: {
    countries: categoriesReducer,
  },
});

export default store;
