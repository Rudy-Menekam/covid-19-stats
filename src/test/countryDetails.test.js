import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import Categories from '../components/categories';
import store from '../redux/store';

describe('Test for countryDetails', () => {
  test('should render', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <Categories />
      </Provider>,
    );
    expect(tree).toMatchSnapshot();
  });
});
