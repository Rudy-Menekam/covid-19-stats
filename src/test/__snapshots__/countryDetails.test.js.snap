// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for countryDetails should render 1`] = `
Array [
  <div
    className="section"
  >
    <form
      className="search"
    >
      <input
        onChange={[Function]}
        placeholder="Search here..."
        required={true}
        type="search"
        value=""
      />
      <button
        onClick={[Function]}
        type="submit"
      >
        Search
      </button>
    </form>
  </div>,
  <div
    className="grid-container"
  />,
]
`;
